NodeManagerLibrary.ino

```diff
@@ -2662,6 +2662,77 @@ int SensorMHZ19::_readCO2() {
 
 #endif
 
+/*
+   SensorSDS011
+*/
+#ifdef USE_SDS011
+SensorSDS011::SensorSDS011(NodeManager & node_manager, int rxpin, int txpin, int child_id): Sensor(node_manager, rxpin){
+  _name = "SDS011";
+  _rx_pin = rxpin;
+  _tx_pin = txpin;
+  children.allocateBlocks(2);
+  new ChildFloat(this, _node->getAvailableChildId(child_id), S_AIR_QUALITY, V_LEVEL, _name);
+  new ChildFloat(this, _node->getAvailableChildId(child_id+1), S_AIR_QUALITY, V_LEVEL, _name);
+}
+
+// Set sensor sleeping
+void SensorSDS011::setSleep(bool value){
+  _slp = value;
+}
+
+// what to do during setup
+void SensorSDS011::onSetup(){
+  _sds = new SDS011();
+  _sds->begin(_rx_pin, _tx_pin);
+  delay(2000);
```

Comment:
Please use wait() instead of delay() throughout the code since delay() will prevent the node to receive messages from the radio network